turtl_backup/turtl_backup_gui.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env python3
 
 import sys
+
```

Comment:
As you dropped cxFreeze, this file is now useless, please drop it completly (and hook the entry point directly to the main of gui.py).

Reply:
fix 91ca6fa9a16723fef9624d27d1556e4f5777610b